h2o-core/src/main/java/water/api/FeatureFlagHandler.java

```diff
@@ -0,0 +1,26 @@
+package water.api;
+
+import water.MRTask;
+import water.api.schemas3.FeatureFlagV3;
+import water.rapids.Rapids;
+import water.rapids.Val;
+
+
+public class FeatureFlagHandler extends Handler {
+  
+
+  @SuppressWarnings("unused") // called via reflection
+  public FeatureFlagV3 setFeatureFlag(int version, FeatureFlagV3 request) throws Exception {
+    Val val = Rapids.exec("(setproperty 'sys.ai.h2o." + request.property + "' '" + request.value + "')");
+    new MRTask() {
```

Comment:
do you really need this task? It seems to be just verifying that `Rapids.exec` did its job, so I find this additional precaution a bit surprising.